event/event.go

```diff
@@ -192,7 +192,7 @@ func (ev *event) Start(ctx context.Context) error {
 	}
 	for _, team := range ev.store.GetTeams() {
 
-		ev.teamQueue <- team
+		ev.teamQueue <- &team
```

Comment:
👍🏻 actually, we do not need to make changes teamQueuening thing.

Reply:
But if it will be the case, then should not we have teams who shares same solutions and answers ? I mean, for example if team-a is signed up and solved 2 challenges then newly registered team-b will have 2 challenges solved (shares same results with team-a) when they registered.  Do you mean that ?

Reply:
No i don't think that's an issue.

```
Imagine there are 3 teams in the ev.store.GetTeams(). 

for team = team1:
    process runs and assigns a lab to team = team1
for team = team2:
    no labs so pointer to `team` is added to queue
for team = team3:
    no labs so pointer to `team` is added to queue

So now the queue contains to pointers both pointing variable `team` which is team3.

When labs are made instead of assigning them to team2 then team3, they are assigned to team3 and team3.
```

Reply:
So, when the number of available labs < number of teams, then we will have a situation where  a team will have multiple labs ?

Reply:
Yes, we could also wait for them to be available like (refactor-labhub)[https://github.com/aau-network-security/haaukins/blob/refactor-labhub/event/event.go#L189].

However that's a bit clunky to do in the current labhub implementation.

Reply:
or we can alter and merge refactor-labhub first then.

Reply:
Yeah, why was the pull request closed?

Reply:
https://github.com/aau-network-security/haaukins/pull/245